### PR TITLE
improve makefiles

### PR DESCRIPTION
**Pull request by cedounet**
This brings a couple of things

TL;DR should be faster (less LaTeX and biber run) and more reliable in your day to day work.

Latexmk handles all dependencies and rerun of Biber etc.

does not force a rebuild by cleaning before build

you can build ebooks and pdf serif or not independently 

no more sed to build the accessible version 

a clean target will remove intermediate files keeping pdf and ebooks.  Mrproper will delete everything 

help and default rules in makefile

note that TikZ are only built with serif… not sure how much of an issue that is.

does not build in parallel either (was not before anyway) mainly due to ebooks in various format competing for aux file

### CODE REVIEW

**Comment by cedounet** [comment]
Hold on, need to check something first...

**Comment by hendricius** [comment]
Nice PR with a lot of great changes! Let me know when you'd like me to have a look and review.

**Comment by cedounet** [comment]
I think we are good.  Not perfect but better anyway.  Please review and let me know any comments/concerns.

**Comment by hendricius** [comment]
Thank you! Great improvements. I don't understand the full magic behind the makefile changes yet - but I will figure it out eventually. Nicely done!
